nifi-extension-bundles/nifi-extension-utils/nifi-listed-entity/src/main/java/org/apache/nifi/processor/util/list/AbstractListProcessor.java

```diff
@@ -229,6 +229,8 @@ protected enum ListingMode {
             .identifiesControllerService(RecordSetWriterFactory.class)
             .build();
 
+    public static final String OBSOLETE_DISTRIBUTED_CACHE_SERVICE = "Distributed Cache Service";
```

Comment:
It seems better to avoid introducing public static variables that are only referenced in tests. Although it can be useful for testing, I would recommend avoiding it for the purpose of testing migrations.